docs/resources/service_compute.md

```diff
@@ -411,9 +411,12 @@ Required:
 
 - **name** (String) The unique name of the Google Cloud Pub/Sub logging endpoint. It is important to note that changing this attribute will delete and recreate the resource
 - **project_id** (String) The ID of your Google Cloud Platform project
-- **secret_key** (String) Your Google Cloud Platform account secret key. The `private_key` field in your service account authentication JSON
 - **topic** (String) The Google Cloud Pub/Sub topic to which logs will be published
-- **user** (String) Your Google Cloud Platform service account email address. The `client_email` field in your service account authentication JSON
+
+Optional:
```

Comment:
So this was actually quite interesting to me, so I raised a query with the official terraform documentation maintainers:

https://github.com/hashicorp/terraform-plugin-docs/issues/42

NOTE: this isn't a blocker to the PR getting approved/merged (as it's outside the control of both the contributor and ourselves as maintainers of this terraform provider), it's just something I noticed and thought I'd share.

Reply:
One thing worth noting here is that the field does in fact become optional from the perspective of the user, since if you don't explicitly pass it then it will default to `""` (empty string). In practice, we can leave `secret_key` out of the TF config block entirely.

But I do see your point that since it will always have a value, and is required, it is a little strange to call it "optional" in the documentation.